src/openzaak/components/zaken/tests/admin/test_zaak_admin.py

```diff
@@ -297,3 +299,28 @@ def test_filter_on_betrokkene_rsin(self):
         identificatie_element = result_list.find(class_="field-identificatie")
 
         self.assertEqual(identificatie_element.text, rol.zaak.identificatie)
+
+    def test_zaakgeometrie(self):
+        """
+        Tests that lat long coords (ESPG 4326) are swapped within the map.
+
+        The map uses ESPG 3857 so the coords need to transform
+        before they can be checked against the expected value.
+        """
+        lat = 52.38554000854492
+        long = 4.842977046966553
+
+        zaak = ZaakFactory.create(zaakgeometrie=Point(lat, long))
```

Comment:
For my understanding: the GeoJSON spec expects the order `[long, lat]` via the API right? Could you replace the factory with an API call to create a Zaak that way, to make sure that this is shown correctly in the admin?

Reply:
If i  use `[long, lat]` in a api call (or with the Point model) it will be shown incorrectly in the widget. (the same as on main). [see comment](https://github.com/open-zaak/open-zaak/pull/1931#issuecomment-2733259237)

If we want the api to also use long lat, the Content-Crs and Accept-Crs headers may need to be changed?

Reply:
@Floris272 it's probably in that order because postgis seems to expect `long, lat` https://postgis.net/documentation/tips/lon-lat-or-lat-lon/

This also aligns with what GeoJSON expects: https://www.rfc-editor.org/rfc/rfc7946#section-3.1.1

So even though we do use the coordinate system EPSG:4326, since we want to adhere to the GeoJSON standard we should use long, lat as well I think